src/vos/README.md

```diff
@@ -796,6 +796,14 @@ to the client telling it to send the RPC to the leader instead.  If it is the
 leader and is in any state other than 'committed' or 'committable', the entry
 is ignored, and the latest committed modification is returned to the client.
 
+If the object to be read is an EC object, and if the data record to be read on the
+data shard (non-leader) attached with a 'prepared' DTX that may be 'committable'
+on the leader because of above asynchronously batched commit mechanism, then the
+non-leader will refresh related DTX status with the leader. If the DTX status
+after refresh is 'committed', then related data can be returned to the client;
+otherwise, such data record is ignored and the latest committed modification is
+returned to the client.
```

Comment:
is this last sentence actually true in practice?   VOS rules for transactions are to wait for prior writes to resolve

Reply:
That is stale. I will fix it.